Cisco/cisco-secure-firewall/tests/test_ASA_852001.json

```
@@ -0,0 +1,26 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Cisco Secure Firewall",
+        "dialect_uuid": "466aeca2-e112-4ccc-a109-c6d85b91bbcf"
+      }
+    },
+    "message": "%FTD-6-1234001: Received Lightweight to full proxy event from application Snort for TCP flow 1.2.3.4/10000 to 4.3.2.1/47003"
```

Could you update the test with the event code `852001`?
```suggestion
    "message": "%FTD-6-852001: Received Lightweight to full proxy event from application Snort for TCP flow 1.2.3.4/10000 to 4.3.2.1/47003"
```